### DISCUSSION TITLE
error linking resend with supabase

### DISCUSSION THREAD

**Discussion by AbolAlia** [supabase/supabase#28077] [Questions]
Hi! I woukd ask you about integration resend with supabase, when i've tried it this error shown, what it mean ?!

nb: i'm not a programmer
![image](https://github.com/user-attachments/assets/1c0ca636-f8a2-421e-8a9f-9d21c3de98b4)

**Comment by olifirDD** [comment]
Hello @AbolAlia I had absolutely the same problem and can't troubleshoot it.
Subscribing for this thread.

**Reply by GaryAustin1** [reply]
You all also might want to contact Resend.
